Podcast/EpisodeDetailView.swift

```diff
@@ -0,0 +1,105 @@
+//
+//  EpisodeDetailView.swift
+//  Podcast
+//
+//  Created by Mark Bryan on 2/19/17.
+//  Copyright © 2017 Cornell App Development. All rights reserved.
+//
+
+import UIKit
+
+class EpisodeDetailView: UIView {
+    
+    var episodeArtworkImageView: UIImageView!
+    var seriesTitleLabel: UILabel!
+    var episodeTitleLabel: UILabel!
+    var dateLabel: UILabel!
+    var descriptionLabel: UILabel!
+    var recommendButton: UIButton!
+    
+    let leftMargin: CGFloat = 24
+    let artworkDimension: CGFloat = 45.5
+    let episodeTitleYInset: CGFloat = 81.5
+    let dateLabelYSpacing: CGFloat = 12
+    let descriptionLabelSpacing: CGFloat = 24
+    let recommendButtonSpacing: CGFloat = 22.5
+    let bottomSpacing: CGFloat = 28.5
+        
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+        backgroundColor = .white
+        
+        episodeArtworkImageView = UIImageView(frame: .zero)
+        episodeArtworkImageView.frame.origin = CGPoint(x: leftMargin, y: leftMargin)
+        episodeArtworkImageView.frame.size = CGSize(width: artworkDimension, height: artworkDimension)
+        addSubview(episodeArtworkImageView)
+        
+        seriesTitleLabel = UILabel(frame: .zero)
+        seriesTitleLabel.font = UIFont.systemFont(ofSize: 16)
+        seriesTitleLabel.textColor = UIColor.colorFromCode(0x4aaea9)
+        addSubview(seriesTitleLabel)
+        
+        episodeTitleLabel = UILabel(frame: .zero)
+        episodeTitleLabel.font = UIFont.systemFont(ofSize: 24)
+        episodeTitleLabel.lineBreakMode = .byWordWrapping
+        episodeTitleLabel.numberOfLines = 0
+        addSubview(episodeTitleLabel)
+        
+        dateLabel = UILabel(frame: .zero)
+        dateLabel.font = UIFont.systemFont(ofSize: 14)
+        addSubview(dateLabel)
+        
+        descriptionLabel = UILabel(frame: .zero)
+        descriptionLabel.font = UIFont.systemFont(ofSize: 14)
+        descriptionLabel.textColor = UIColor.colorFromCode(0xa8a8b4)
+        descriptionLabel.lineBreakMode = .byWordWrapping
+        descriptionLabel.numberOfLines = 0
+        addSubview(descriptionLabel)
+        
+        recommendButton = UIButton(frame: .zero)
+        recommendButton.titleLabel?.font = UIFont.systemFont(ofSize: 14)
+        recommendButton.setTitleColor(UIColor.colorFromCode(0xa8a8b4), for: .normal)
+        recommendButton.setTitle("Recommend", for: .normal)
+        recommendButton.sizeToFit()
+        recommendButton.frame.origin.x = leftMargin
+        recommendButton.addTarget(self, action: #selector(recommendTapped), for: .touchUpInside)
+        addSubview(recommendButton)
+        
+        updateUI()
+    }
+    
+    func updateUI() {
+        episodeArtworkImageView.image = #imageLiteral(resourceName: "sample_series_artwork")
```

Comment:
this is fine for now, but we'll need to load this asynchronously using an imageURL later